src/smart-components/role/add-role-new/add-role-wizard.scss

```diff
@@ -40,4 +40,20 @@
     }
 }
 
-
+.ins-c-rbac-permissions-table {
+    .ins-c-primary-toolbar {
+        .pf-c-toolbar__content {
+            .pf-c-toolbar__content-section {
+                .pf-m-filter-group {
+                    .ins-c-primary-toolbar__filter {
+                        .ins-c-conditional-filter {
+                            .pf-c-select {
```

Comment:
Phew, this is really complex. Isn't it possible to simplify it a bit?

Reply:
I didn't want to change the Toolbar component, so there is quite complex selection, but I think I can simplify it a bit.